editor/src/components/canvas/dom-walker.ts

```diff
@@ -610,11 +609,10 @@ function collectMetadata(
       const unfilteredChildrenPaths = allUnfilteredChildrenPaths.concat(rootsOrChildrenToAdd)
 
       let filteredChildPaths: ElementPath[] = []
-      let filteredRootElements: ElementPath[] = []
       fastForEach(unfilteredChildrenPaths, (childPath) => {
         if (EP.isParentOf(path, childPath)) {
           if (EP.isRootElementOfInstance(childPath)) {
-            filteredRootElements.push(childPath)
+            // Delete this when killing child paths
```

Comment:
Is this something that needs fixing imminently or for functionality we don't currently have?

Reply:
so this is just an empty IF branch, which I should have flipped. However, (spoilers) I plan to do the same thing to ElementInstanceMetadata.children, which means I'll delete this entire block